### PR TITLE
Login layout#769

### LINKED ISSUE #769: Sign-in/Login page layout
**Describe the bug**
Login and sing-in page needs layout changes

1. ~~Remember me checkbox is not styled at all~~ (fixed in #1259)
2. Social login buttons look odd when they are aligned that way if we want to stick to circles the best way to fix that will be something like that:

image

3. Elements are not aligned with each other

Bez tytułu

4. Show password icon (eye) should be inside a textbox

5. Main action buttons (Log-in/Sign-in) are definitely too small and don't stand out when they should be the main button on the page they look very "little"

6. The whole section below text boxes looks like an overloaded mess. The big part in that issue also points `2` but, the important issue is that `New User? Sign up` and `Forgot password? Reset your Password` are basically the same font size and shape like normal page elements or titles when those elements are rarely used by end-user. They should be grey and don't stand out that much

7. I described login page mostly here but the sign-in page has exactly same problems

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Vivaldi (chromium)

**Are you working on this issue? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by AyanChoudhary**
Fixes #769 

#### Describe the changes you have made in this pr -
1. changed the position of show password icon to inside the text box
2. added styling to remember me checkbox

### Screenshots of the changes (If any) -
circuitverse

### CODE REVIEW

File: app/views/users/sessions/new.html.erb [hunk 1]
```diff
@@ -17,24 +17,24 @@
...
+            <%= f.email_field :email, autofocus: true, class:"border-0 custom-input", placeholder: 'Email'%>
           </div>
         </div>
 
         <div class="field form-group">
           <%= f.label :password, class:"label-text" %>
-          <div class="input-group">
-            <%= f.password_field :password, autocomplete: "off" ,class:"form-control custom-input", placeholder: 'Password',  id:"password-input"%>
+          <div class="input-group form-control custom-input">
+            <%= f.password_field :password, autocomplete: "off" ,class:"border-0 custom-input", placeholder: 'Password',  id:"password-input"%>
```
[the remark this answers is not in the document]
Comment (AyanChoudhary) [reply app/views/users/sessions/new.html.erb]: The spacing is the same as earlier I did not make any changes in that.

File: app/views/users/sessions/new.html.erb [hunk 2]
```diff
@@ -17,24 +17,24 @@
 
         <div class="field form-group">
           <%= f.label :email, class:"label-text" %>
-          <div class="input-group">
-            <%= f.email_field :email, autofocus: true, class:"form-control custom-input", placeholder: 'Email'%>
+          <div class="input-group form-control custom-input">
+            <%= f.email_field :email, autofocus: true, class:"border-0 custom-input", placeholder: 'Email'%>
```
[the remark this answers is not in the document]
Comment (AyanChoudhary) [reply app/views/users/sessions/new.html.erb]: The spacing is the same as earlier I did not make any changes in that.

**Comment by thiefie** [comment]
@AyanChoudhary Please modify existing system tests so that they can work with the newly renamed HTML elements. This should fix the CI build also

**Comment by aayush-05** [comment]
@AyanChoudhary are you still working on this ? @nitin10s shall we close this, no response since a long time

**Comment by nitin10s** [comment]
Closing this:
1. We'll be changing the login design to modal based.
2. Checkboxes need to be worked upon as a single unit wherever they are used across the platform

**Comment by MrBartusek** [comment]
> Closing this:
> 
> 1. We'll be changing the login design to modal based.
> 2. Checkboxes need to be worked upon as a single unit wherever they are used across the platform

shall i close #769?

**Comment by nitin10s** [comment]
Yeah. I hope all the problems you mentioned have been rectified in the new design.

**Comment by MrBartusek** [comment]
@nitin10s well, the issue is still valid but since you closed that pr because of
> We'll be changing the login design to modal based

That means there is no reason to fix it because it will be changed anyway?

**Comment by nitin10s** [comment]
Yeah, by the previous statement, I meant the problems you mentioned have been rectified in the "new" popup design. Yes close that issue as well.

**Pull request closed without merging**